sdk/monitor/monitor-opentelemetry-exporter/package.json

```diff
@@ -86,15 +85,15 @@
     "@azure/dev-tool": "^1.0.0",
     "@azure/eslint-plugin-azure-sdk": "^3.0.0",
     "@microsoft/api-extractor": "7.18.11",
-    "@opentelemetry/instrumentation": "^0.28.0",
-    "@opentelemetry/instrumentation-http": "^0.28.0",
+    "@opentelemetry/instrumentation": "^0.29.2",
```

Comment:
Any reason to no update other dependencies used in the exporter itself? 1.3.1 is available for:
@opentelemetry/core, @opentelemetry/resources, @opentelemetry/sdk-trace-base and @opentelemetry/semantic-conventions

Reply:
Our bot only logs issues for available major version updates.  In general, we assume that with the caret version ^1.2.0 customers should be able to get the latest 1.x.x without code changes on our side. Of course there are cases where we bump versions too, for example, to enforce certain minimum version in our dependencies for security fixes .

Reply:
@hectorhdzg Those dependencies with minor or path version updates automatically get updated by rush, as it's satisfied with the semver versioning ^. If you see the pnpm lock file in the PR has the updates. No need to change in package.json.